optimade/filtertransformers/mongo.py

```diff
@@ -232,9 +228,13 @@ def set_zip_op_rhs(self, arg):
         # ANY value_zip_list )
         raise NotImplementedError
 
-    def length_comparison(self, arg):
-        # length_comparison: LENGTH property OPERATOR value
-        raise NotImplementedError
+    def predicate_comparison(self, arg):
```

Comment:
Ah, you've already re-implemented `predicate_comparison` here.

So I'm confused. How does this link up with the current un-merged [PR](https://github.com/Materials-Consortia/OPTiMaDe/pull/191)?